src/handlers/payout/action.ts

```diff
@@ -88,6 +89,22 @@ export const handleIssueClosed = async () => {
   }
 
   logger.info(`Handling issues.closed event, issue: ${issue.number}`);
+
+  for (const botComment of comments.filter((cmt) => cmt.user.type === "Bot").reverse()) {
```

Comment:
We have a global variable somewhere for `Bot`, I pretty much don't like using hardcoded values.